libs/common/src/auth/services/device-trust-crypto.service.implementation.ts

```diff
@@ -22,19 +26,38 @@ import {
   UpdateDevicesTrustRequest,
 } from "../models/request/update-devices-trust.request";
 
+/** Uses disk storage so that the device key can persist after log out and tab removal. */
+export const DEVICE_KEY = new KeyDefinition<DeviceKey>(DEVICE_TRUST_DISK_LOCAL, "deviceKey", {
+  deserializer: (deviceKey) => SymmetricCryptoKey.fromJSON(deviceKey) as DeviceKey,
+});
+
+/** Uses disk storage so that the shouldTrustDevice bool can persist across login. */
+export const SHOULD_TRUST_DEVICE = new KeyDefinition<boolean>(
+  DEVICE_TRUST_DISK_LOCAL,
+  "shouldTrustDevice",
+  {
+    deserializer: (shouldTrustDevice) => shouldTrustDevice,
```

Comment:
Question: Do we want to return nullish from here?
```suggestion
    deserializer: (shouldTrustDevice) => shouldTrustDevice ?? false,
```

Reply:
Discussed synchronously; leaving as is.